app/src/main/java/com/example/benchmarks/domain/models/OperationItem.java

```diff
@@ -0,0 +1,19 @@
+package com.example.benchmarks.domain.models;
+
+import com.example.benchmarks.domain.models.operation.Operation;
+
+public class OperationItem<O extends Operation> {
```

Comment:
@Natttaallia here the generics is not required. 
OperationItem works only with operation type(ofcourse with different inherited operation class) and no other types. 
We use generics when we need to work with different data types, but here we are working only with the operation data type.